include/faabric/scheduler/Scheduler.h

```diff
@@ -9,88 +9,85 @@
 
 #include <shared_mutex>
 
-#define AVAILABLE_HOST_SET "available_nodes"
+#define AVAILABLE_HOST_SET "available_hosts"
```

Comment:
The term `node` was really confusing, so I've now used Faaslet or host where relevant